core/server/master/src/test/java/alluxio/master/file/meta/AsyncUfsAbsentPathCacheTest.java

```diff
@@ -317,4 +334,18 @@ private void checkPaths(AlluxioURI firstAbsent) throws Exception {
       existing = existing.getParent();
     }
   }
+
+  static class TestAsyncUfsAbsentPathCache extends AsyncUfsAbsentPathCache {
```

Comment:
nit: add a comment why this class is needed

Reply:
added in 27abf0c3